written_about_us/kindey_exchange_israel.json

```diff
@@ -1,4 +1,39 @@
 {
+  "best_articles": [
+    {
+      "date": "21.10.2020",
+      "title": "V IKEM provedli další párovou výměnu ledvin. Lékaři v Česku a Izraeli nepracovali simultánně",
+      "url": "https://www.irozhlas.cz/zivotni-styl/zdravi/ikem-cesko-izrael-parova-transplantace-ledvin_2010221143_miz",
+      "img": "https://www.irozhlas.cz/sites/default/files/styles/zpravy_fotogalerie_large/public/uploader/img_0104_170426-015347_dp.jpg",
```

Comment:
obrazky pojdme stahnout a fouknout sem do repa do slozky assets, buhvi jakou budou mit tyhle weby retenci :)

Reply:
a pak teda hodit URL z repa sem

Reply:
no nevim, kdyz ten clanek zmizi, tak prestane mit smysl i ten link a vlastne muzeme rovnou celej ten odkaz smazat ne?

Reply:
hah, to je vlasten dobrej argument no, to mi nenapadlo, beztak na obrazkach nejsou nase fotky, takze asi ano no, pokud zmizi clanek, tak na to nema smysl odkazovat 

jo, nechme to teda takhle